frontend/src/components/Header/Header.tsx

```diff
@@ -18,6 +18,7 @@ const StyledTopBar = styled(TopBar)`
     @media (max-width: 600px) {
         grid-column-gap: 12px;
     }
+    height: fit-content;
```

Comment:
Queue mission button becomes squashed

Before:
<img width="469" alt="image" src="https://github.com/user-attachments/assets/11ccbffa-cc23-4a9e-8cfb-5aa5ae3978fd" />
In this PR:
<img width="450" alt="image" src="https://github.com/user-attachments/assets/49377c6f-ff35-41bb-a2be-277bb1947785" />